authentication-request-handler/pom.xml

```diff
@@ -0,0 +1,170 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Please add the license header.

Reply:
Fixed with c907fcc2a908e093c988f8bf86c54ce8e98b72c0